src/defaults.js

```diff
@@ -19,14 +17,4 @@ function getDefaults() {
     smartypants: false,
     xhtml: false
   };
-}
-
-function changeDefaults(newDefaults) {
-  defaults = newDefaults;
```

Comment:
It looks like this should actually be `module.exports.defaults = newDefaults;`

Reply:
I think this worked as it is when the files were ES modules but when I switched to CommonJS I forgot to update this line.

Reply:
This could have happened because ESM exports are actually live bindings but it doesn't work like that with CJS (as you have noticed you'd have to assign it to module.exports to make it work).